src/main/java/org/matsim/analysis/preanalysis/PopulationAnalysis.java

```diff
@@ -0,0 +1,83 @@
+package org.matsim.analysis.preanalysis;
+
+import org.locationtech.jts.geom.Geometry;
+import org.matsim.api.core.v01.Coord;
+import org.matsim.api.core.v01.Scenario;
+import org.matsim.api.core.v01.population.Activity;
+import org.matsim.api.core.v01.population.Person;
+import org.matsim.api.core.v01.population.PlanElement;
+import org.matsim.api.core.v01.population.Population;
+import org.matsim.core.config.Config;
+import org.matsim.core.config.ConfigUtils;
+import org.matsim.core.scenario.ScenarioUtils;
+import org.matsim.core.utils.gis.ShapeFileReader;
+import org.opengis.feature.simple.SimpleFeature;
+
+import java.io.FileWriter;
+import java.io.IOException;
+import java.util.Collection;
+
+public class PopulationAnalysis {
```

Comment:
Would be great if this class would use the app command functionality and options instead of hard coded paths